spk/sabnzbd/Makefile

```diff
@@ -18,7 +18,7 @@ DESCRIPTION_SPN = SABnzbd hace que Usenet sea lo más simple posible, automatiza
 RELOAD_UI = yes
 DISPLAY_NAME = SABnzbd
 STARTABLE = yes
-CHANGELOG = "Update SABnzbd to 3.0.2. Please note that SABnzbd now runs on Python 3. Make sure your post/pre-processing scripts work with Python 3!"
+CHANGELOG = "Update SABnzbd to 3.1.0 and include SabToSyno.py script to add downloaded files automatically to the Synology DLNA server. Please note that SABnzbd now runs on Python 3. Make sure your post/pre-processing scripts work with Python 3!"
```

Comment:
I guess you mean "DNLA" server

Reply:
I would move "Please note that SABnzbd now runs on Python 3. Make sure your post/pre-processing scripts work with Python 3!" to install/upgrade wizard or wiki dedicated page FAQ at https://github.com/SynoCommunity/spksrc/wiki/Package-Documentation-Index

Reply:
Remove the Python 3 section, since previous package was also already Python 3 and we didn't get any complaints.
DLNA is the correct term as far as I know: https://www.synology.com/en-global/knowledgebase/DSM/tutorial/Multimedia/How_to_enjoy_multimedia_contents_stored_on_Synology_NAS_with_DLNA_UPnP_compliant_DMAs